IPlug/IPlugConstants.h

```diff
@@ -29,7 +34,10 @@ enum EIPlugKeyCodes
 
 //TODO: these should be in a namespace, to avoid conflicts with third-party libraries
 static const double PI = 3.141592653589793238;
+
+// Magic number. Approximates 20 * log10(x) for gain to dB conversion
 static const double AMP_DB = 8.685889638065036553;
+// Reverse conversion
```

Comment:
This approximates 10^(x/20) as seen here: http://www.wolframalpha.com/input/?i=plot+exp(0.11512925464970+*+x),+10^(x%2F20)